### PR TITLE
Feature/user details

### PR DESCRIPTION
**Pull request by whyDontI**
Added
- Update user route

### CODE REVIEW

File: user/README.md [hunk 1]
```diff
@@ -1,80 +1,126 @@
...
+	'company': string,,
+	'designation': string,
+	'img': string,
+	'github_id': string,
+	'linkedin_id': string,
+	'twitter_id': string,
+  'instagram_id': string,
+  'site': string,
+  'github_display_name': string,
+  'isMember': boolean
```
Comment (ankushdharkar) [inline user/README.md:20]: Please fix formatting

File: user/README.md [hunk 2]
```diff
@@ -1,80 +1,126 @@
...
 }
 ```
 
-## **GET /member**
+## **GET /user**
 
-Returns all members in the system.
+Returns all users in the system.
```
Comment (ankushdharkar) [inline user/README.md:26]: Then is should be ideally called `/users`

File: user/README.md [hunk 3]
```diff
@@ -1,80 +1,126 @@
...
   **Content:**
 
 ```
 {
-  message: "Members returned successfully!"
-  members: [
-           {<member_object>},
-           {<member_object>},
-           {<member_object>}
+  message: 'Users returned successfully!'
```
Comment (ankushdharkar) [inline user/README.md:44]: Let's call it 'msg' to keep it consistent with the other API

**Review by ankushdharkar — CHANGES_REQUESTED** [review]

File: user/README.md (hunk 3 above)
Comment (whyDontI) [reply user/README.md:44]: I case of error i.e. 4XX
if returns `{ error: 'Not Found', message: 'No users available' }`
which is being handled by boom (an npm package) so I thought I'll keep it same.

Comment (ankushdharkar) [reply user/README.md:44]: Great point. Then let's change the other one to "message" in the future

File: user/README.md [hunk 4]
```diff
@@ -57,7 +57,7 @@ Returns all users in the system.
   - **Code:** 401  
     **Content:** `{ 'statusCode': 401, 'error': 'Unauthorized', 'message': 'Unauthenticated User' }`
 
-## **GET /user/:id**
+## **GET /users/:id**
```
Comment (ankushdharkar) [inline user/README.md:60]: This should be singular 'user'

File: user/README.md [hunk 5]
```diff
@@ -102,7 +102,7 @@ Creates a new User.
   - **Code:** 401  
     **Content:** `{ 'statusCode': 401, 'error': 'Unauthorized', 'message': 'Unauthenticated User' }`
 
-## **PATCH /user**
+## **PATCH /users**
```
Comment (ankushdharkar) [inline user/README.md:105]: 'user'

**Review by ankushdharkar — CHANGES_REQUESTED** [review]
Please call singular returns as 'user'

File: user/README.md (hunk 4 above)
Comment (whyDontI) [reply user/README.md:66]: Agreed upon `/users`

**Comment by whyDontI** [comment]
Please merge If no changes required

**Comment by ankurnarkhede** [comment]
LGTM.

**Pull request merged**